### PR TITLE
Update Readme.md and add paypal.me donate link (#2250)

### LINKED ISSUE #2250: Add paypal.me link to Readme for project donations
We currently carry a prominent "Donate" main menu item within the Web-UI. Given this link has been used a total of once in the last couple of years, and it's reliance on an external link (the only one within our initial Web-UI load), that of a:

GET | https://www.paypalobjects.com/en_GB/i/btn/btn_donate_SM.gif

and it's use of a now 'legacy' PayPal button arrangement; it is proposed that we add a PayPal.me link to our Readme.md file as this may serve as either an augmentation to the existing Web-UI donate elements or more likely as a replacement/alternative target given the relative 'usability/space value' of a Web-UI element and the proven lack of use to date.

Post merge, pending review, there is the option to use our main GitHub Readme.md file to 'carry' this donate element; a common practice in open source projects. And the further possibility of removing the Donate elements from within the Web-UI entirely without completely removing the donation mechanisms/links entirely from the project as a whole.

The following non legacy PayPal link is proposed for Readme addition:
https://www.paypal.me/rockstor

### PR DESCRIPTION
**Pull request by phillxnet**
Revised Readme.md so that it more closely matches our far more recent rockstor.com contents.

Fixes #2250 
@FroggyFlox Ready for review.
As indicated in the referenced issue, I'd like to get this one in before embarking on some revisions of the Web-UI re the donate mechanism. That way we at least have a non legacy Paypal donate mechanism in place. The other text edits represent a scope creep but hopefully an acceptable one :) .

Summary of changes:
- Removed legacy build status link.
- Added Donate link.
- Updated to more closely reflect current rockstor.com contents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,74 @@
-# Rockstor
+# The Rockstor Project
 
-[![Build Status](http://192.195.81.208:22082/buildStatus/icon?job=build-rockstor-core)](http://192.195.81.208:22082/job/build-rockstor-core)
+An open source (GPL) community endeavour to sustainably develop, maintain, and
```
Comment (StephenBrown2) [inline README.md:3]: Any particular version of GPL specified? i.e. v2? v3?

File: README.md [hunk 2]
```diff
@@ -1,63 +1,74 @@
...
-the current release, it's intended for advanced home users, DIY NAS builders
-and smaller organizations that need NAS or private cloud in a box. Go to
-[rockstor.com](http://rockstor.com) for more information.
+Rockstor aims to aid individuals and organisations alike. As a DIY setup some
+familiarity with PC/Pi4 operating system install is required; but not much beyond
+the basics. For more information please see our main web site:
+[rockstor.com](http://rockstor.com)
 
-# Getting started
+# Getting started & any questions
```
Comment (StephenBrown2) [inline README.md:34]: Special characters may mess up anchor linking
```suggestion
# Getting started and questions
```

File: README.md [hunk 3]
```diff
@@ -1,63 +1,74 @@
...
-[community forum](http://forum.rockstor.com).
+To get in touch with developers, users, and contributors join our
+[community forum](http://forum.rockstor.com) and ask away.
 
 # Documentation
 
-Documentation is in the
-[rockstor-doc](https://github.com/rockstor/rockstor-doc) repository. You can also read it
-[online](http://rockstor.com/docs).
+[Our Documentation](http://rockstor.com/docs) is also a community endeavour with it's
```
Comment (StephenBrown2) [inline README.md:44]: I often get messed up by this as well.
```suggestion
[Our Documentation](http://rockstor.com/docs) is also a community endeavour with its
```

File: README.md [hunk 4]
```diff
@@ -1,63 +1,74 @@
...
+own GitHub repo: [rockstor-doc](https://github.com/rockstor/rockstor-doc): contributions
+and correction are always welcome.
```
Comment (StephenBrown2) [inline README.md:46]:
```suggestion
and corrections are always welcome.
```

File: README.md [hunk 5]
```diff
@@ -1,63 +1,74 @@
...
 Please see the file [COPYING](https://github.com/rockstor/rockstor-core/blob/master/COPYING).
 
 # Contributing
 
-Development environment setup and contribution guidelines are explained
-[here](http://rockstor.com/docs/contribute.html).
+Development environment setup and contribution guidelines are available in our docs:
+[Contributing to Rockstor - Overview](http://rockstor.com/docs/contribute.html)
+
+There is no contributor licence agreement.
```
Comment (StephenBrown2) [inline README.md:69]: Spelling (may be a US/UKism), and maybe capitalization?
```suggestion
There is no Contributor License Agreement.
```

File: README.md [hunk 6]
```diff
@@ -1,63 +1,74 @@
...
 
-Pull requests are welcome. Pick an issue that interests you or create a new one
-to work on. You can also ask on the [community forum](http://forum.rockstor.com) for
-guidance. We'll be happy to help.
+Pull requests are welcome. Pick an issue that interests you or create a new one.
+Please also references our [friendly community forum ](http://forum.rockstor.com)
```
Comment (StephenBrown2) [inline README.md:72]:
```suggestion
Please also reference our [friendly community forum ](http://forum.rockstor.com)
```

File: README.md [hunk 7]
```diff
@@ -1,63 +1,74 @@
...
+for open questions and our ever growing wiki entries that serve as our
```
Comment (StephenBrown2) [inline README.md:73]:
```suggestion
for open questions and our ever-growing wiki entries that serve as our
```

**Comment by FroggyFlox** [comment]
@phillxnet, thanks for updating this, it does look a lot better, indeed.

I agree with @StephenBrown2's suggestions (thanks a lot!) and do not have any additional one besides those.

File: README.md (hunk 4 above)
Comment (phillxnet) [reply README.md:46]: @StephenBrown2 OK, thanks. This was a silly mistake on my part.
Cheers.

File: README.md (hunk 5 above)
Comment (phillxnet) [reply README.md:69]: @StephenBrown2 Thanks again. Yes UK/US spelling but lets go with this change as it makes no odds and the capitalisation with accompanying TLA is nice.

File: README.md (hunk 6 above)
Comment (phillxnet) [reply README.md:72]: Thanks. Nice catch. Sill me.

File: README.md (hunk 7 above)
Comment (phillxnet) [reply README.md:73]: Agreed. @FroggyFlox over time and many reviews has warmed to to the use of the hyphen for readability.

**Comment by phillxnet** [comment]
@StephenBrown2 and @FroggyFlox  Thanks for the feedback. Much appreciated.
I'll push another commit for that GPL version suggestion. I'm afraid I tried to fit this seemingly quick pr into what has turned out to be an insufficient time slot. Oh well. All the better for your feedback. Apologies for the rough initial state, but we are getting there now.

The GPL version was a hovering point for me as although our core repo is GPLv3 as per the License / COPYING file references some of our runtime third party libraries are more v2 compatible than v3. But in this case we are referencing this repo I think so lets go with v3 to be consistent with the later reference in the same file at least.

**Comment by phillxnet** [comment]
@StephenBrown2 and @FroggyFlox Thanks again for your input on this one.

Is the consensus that we are now good-to-go on this one? If so I can merge and move on to cleaning up our Web-UI donate references to remove the now legacy Paypal button references.

**Comment by FroggyFlox** [comment]
Looks good to me, thanks!

**Comment by phillxnet** [comment]
Thanks folks. I'll go ahead and merge this one. We can always re-visit and I think we have at least nudged it towards being a little  better than it was.

**Pull request merged**